src/sidebar/util/test/fetch-config-test.js

```diff
@@ -237,6 +239,52 @@ describe('sidebar.util.fetch-config', () => {
         const result = fetchConfig(appConfig, fakeWindow);
         return assertPromiseIsRejected(result, 'Nope');
       });
+
+      it('returns the `groups` array with the initial host config request', async () => {
+        const appConfig = {
+          services: [{ groups: ['group1', 'group2'] }],
+          appType: 'via',
+        };
+        fakeJsonRpc.call.onFirstCall().resolves({ foo: 'baz' }); // host config
+        const result = await fetchConfig(appConfig, fakeWindow);
+        assert.deepEqual(await result.services[0].groups, ['group1', 'group2']);
```

Comment:
Is there a need to `await` the groups array here? It should be a plain array in this case shouldn't it?